azure/packages/azure-client/src/AzureClient.ts

```diff
@@ -101,11 +158,15 @@ export class AzureClient {
 
 	/**
 	 * Creates new detached container out of specific version of another container.
+	 *
 	 * @param id - Unique ID of the source container in Azure Fluid Relay.
 	 * @param containerSchema - Container schema used to access data objects in the container.
 	 * @param version - Unique version of the source container in Azure Fluid Relay.
-	 * It defaults to latest version if parameter not provided.
+	 * @defaultValue latest. It defaults to latest version if parameter not provided.
```

Comment:
Unfortunately, `@defaultValue` does not work under a param tag. All of these sorts of tags are applied to the item itself, so this comment would be applied as the default value of `copyContainer`, rather than the `version` parameter. I would probably just revert this particular change.